pjsip/include/pjsua2/account.hpp

```diff
@@ -706,9 +706,27 @@ struct AccountNatConfig : public PersistentObject
      * Default constructor
      */
     AccountNatConfig() : sipStunUse(PJSUA_STUN_USE_DEFAULT),
-			 mediaStunUse(PJSUA_STUN_USE_DEFAULT),
-			 nat64Opt(PJSUA_NAT64_DISABLED),
-			 turnConnType(PJ_TURN_TP_UDP)
+      mediaStunUse(PJSUA_STUN_USE_DEFAULT),
+      nat64Opt(PJSUA_NAT64_DISABLED),
+      iceEnabled(false),
+      iceTrickle(PJ_ICE_SESS_TRICKLE_DISABLED),
+      iceMaxHostCands(-1),
+      iceAggressiveNomination(true),
+      iceNominatedCheckDelayMsec(PJ_ICE_NOMINATED_CHECK_DELAY),
+      iceWaitNominationTimeoutMsec(ICE_CONTROLLED_AGENT_WAIT_NOMINATION_TIMEOUT),
+      iceNoRtcp(false),
+      iceAlwaysUpdate(true),
+      turnConnType(PJ_TURN_TP_UDP),
+      contactRewriteUse(true),
+      contactRewriteMethod(PJSUA_CONTACT_REWRITE_METHOD),
+      contactUseSrcPort(PJ_TRUE),
+      viaRewriteUse(PJ_TRUE),
+      sdpNatRewriteUse(PJ_FALSE),
+      sipOutboundUse(PJ_TRUE),
+      sipOutboundInstanceId(""),
+      sipOutboundRegId(""),
```

Comment:
No need to init string members with empty string (as done in some other string fields)?

Reply:
Makes sense, removed